features/counter.feature

```diff
@@ -1,25 +1,26 @@
-Uses: StepRegistry
+Uses: Web Sockets
 Uses: Selenium
 
 Feature: I can use the counter
 
-  #! StepRegistry start
   Feature-Start:
-    Given I open Chrome
+    Given I start a web socket server
+    And I open Chrome
+    And I navigate to http://localhost:9999
+    And I wait for the web socket client SimpleStepPublisher
 
   Background:
-    Given I navigate to http://localhost:9999
-    And StepRegistry client SimpleStepPublisher is connected
+    Given I reset the counter
```

Comment:
Oh I see, you've added this to reset the state instead in the background section?